### PR TITLE
Regression Fix - Content shifted under layered nav on PLP

### PR DESCRIPTION
**Pull request by JustinConabreeAbsolunet**
## Description

Fixing regression introduced by [Layered Navigation](https://github.com/magento/pwa-studio/pull/3137) where content shifts below the layered nav if an item's title exceeds the space available.

## Related Issue

Fixes issue https://jira.corp.magento.com/browse/PWA-1897

### Verification Stakeholders

@revanth0212 

#### Test scenario(s) for direct fix/feature

- Validate that "Shop the look" category displays correctly when browser width is between 1024px - 1100px
- Validate that layout displays correctly outside of this range as well

#### Is Browser/Device testing needed?

Yes, device testing is needed as layered nav + gallery UI components may be impacted on mobile and desktop

## Breaking Changes (if any)

Layout/targetable references of category content changed.

## Checklist

- [x] I have added tests to cover my changes, if necessary.

### CODE REVIEW

**Review by tjwiebell — APPROVED** [review]
Straight forward fix, content responsiveness is very smooth 👍  nice work.

File: packages/venia-ui/lib/RootComponents/Category/category.css [hunk 1]
```diff
@@ -50,14 +50,18 @@
...
 .sidebar {
     display: none;
 }
 
 @media (min-width: 1024px) {
-    .root {
+    .contentWrapper {
         display: flex;
-        flex-wrap: wrap;
+        flex-wrap: nowrap;
```
Comment (tjwiebell) [inline packages/venia-ui/lib/RootComponents/Category/category.css:64]: Tiny nitpick, but this should be the default so this line is redundant; consider removing.

```suggestion
```

Comment (JustinConabreeAbsolunet) [reply packages/venia-ui/lib/RootComponents/Category/category.css]: @tjwiebell Good find! I've removed the value and it's still working.

**Review by eug123 — APPROVED** [review]

**Comment by eug123** [comment]
✅  QA Approved

QA Verification Steps PWA-1897

**Pull request merged**